pkg/network/cloud_network.go

```diff
@@ -20,7 +20,7 @@ import (
 // renderCloudNetworkConfigController renders the cloud network config controller
 func renderCloudNetworkConfigController(conf *operv1.NetworkSpec, cloudBootstrapResult bootstrap.InfraStatus, manifestDir string) ([]*uns.Unstructured, error) {
 	pt := cloudBootstrapResult.PlatformType
-	if !(pt == v1.AWSPlatformType || pt == v1.AzurePlatformType || pt == v1.GCPPlatformType) {
+	if !(pt == v1.AWSPlatformType || pt == v1.AzurePlatformType || pt == v1.GCPPlatformType || pt == v1.OpenStackPlatformType) {
```

Comment:
You should also NOT render it when Kuryr is configured, it won't work. I think you can check it like this: `conf.DefaultNetwork.Type == operv1.NetworkTypeKuryr`.

Reply:
@dulek Does that now look better to you?